config/locales/views.de.yml

```diff
@@ -425,7 +425,7 @@ de:
   event/participations:
     full_entry_label: '%{model_label} von <i>%{person}</i> in <i>%{event}</i>'
     success: '%{full_entry_label} wurde erfolgreich erstellt. Bitte überprüfe die Kontaktdaten und passe diese gegebenenfalls an.'
```

Comment:
Where does `full_entry_label` come from? We would like to change it to "Die Anmeldung" if it currently says "Teilnahme".

Reply:
@fossil12 `full_entry_label` comes from an abstraction layer for a CRUD Controller. See `app/controllers/crud_controller.rb` But as this one is used for multiple models, you should not change it. It's probably best to just create a new decorator method for the participation. See `app/decorators/event/participation_decorator`. You can a new method there `def full_label` which will be available in the views. I also actually think you can remove the part of "bitte überprüfe Kontaktangaben" as not the registration automatically asks for contact details.

Reply:
Thank you for details explanation. The impact of this change seems to big to me do any modification right now.